user_app.go

```diff
@@ -20,7 +20,7 @@ import (
 	"errors"
 	"math"
 
-	"github.com/cosmos/ledger-go"
+	"github.com/zondax/ledger-go"
```

Comment:
should people be importing cosmos or zondax here?

Reply:
`ledger-go` is a library that we actually use for a large amount of projects. It is 100% generic and allows connecting to ledger devices from golang.

We maintain that library much more frequently. If you agree, I would keep this as Zondax.
Would that be fine?

Reply:
oh amazing!!